### ISSUE TITLE
easier way to get pretty printing of Metrics

### ISSUE THREAD

**Issue by antonkulaga** [bigdatagenomics/utils#105]
Wrinting things like
```
val writer = new PrintWriter(new OutputStreamWriter(System.out))
```
is very cumbersome. And it often fails when you run it in SparkNotebook. Why not just giving Seq[String] that can be printed with mkString("\n") if needed?

**Comment by antonkulaga** [comment]
I've tried to write a simple function that returns string but it did not work
```
scala
  def prettyPrint() = {
    import java.io.{PrintWriter, StringWriter}
    val stringWriter = new StringWriter()
    val writer = new PrintWriter(stringWriter)
    val result = stringWriter.toString
    writer.close()
    result
  }
```

**Comment by fnothaft** [comment]
Hi @antonkulaga! I've opened a proposed fix as PR #112. Would love your thoughts on it!
